initial_data.json

```diff
@@ -0,0 +1,92 @@
+[
+    {
+        "model": "alexandria_core.category",
+        "pk": "alle-beteiligten",
+        "fields": {
+            "created_at": "2020-08-04T13:24:00.704Z",
+            "created_by_user": null,
+            "created_by_group": null,
+            "modified_at": "2020-08-04T13:24:00.704Z",
+            "modified_by_user": null,
+            "modified_by_group": null,
+            "meta": {
+
+            },
+            "name": "{\"en\": \"Alle Beteiligten\", \"de\": \"Alle Beteiligten\"}",
+            "description": "{\"en\": \"\"}",
```

Comment:
As this is our demo data, please be consistent with translations. Most objects here have a name in german and a description in english.

Reply:
I've added Englisch names everywhere